### PR TITLE
PM-1986 - Add Helm Docs on all charts + remove legacy charts

### PR DESCRIPTION
**Pull request by kaozenn**

### CODE REVIEW

File: uptime-service-payloads-scrapper/values.yaml [hunk 1]
```diff
@@ -1,32 +1,41 @@
...
 serviceAccount:
-  # Specifies whether a service account should be created
+  # -- Specifies whether a service account should be created
   create: true
-  # Annotations to add to the service account
+  # -- Annotations to add to the service account
   annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
+  # -- The name of the service account to use.
```
Comment (simisimis) [inline uptime-service-payloads-scrapper/values.yaml:27]: see above multi-line comment

File: mina-rosetta/values.yaml [hunk 2]
```diff
@@ -1,31 +1,41 @@
...
-  # Specifies whether a service account should be created
+  # -- Specifies whether a service account should be created
   create: true
-  # Annotations to add to the service account
+  # -- Annotations to add to the service account
   annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
+  # -- The name of the service account to use.
+  # -- If not set and create is true, a name is generated using the fullname template
```
Comment (simisimis) [inline mina-rosetta/values.yaml:28]: see above multi-line comment

File: liminal-manual/values.yaml [hunk 3]
```diff
@@ -2,36 +2,47 @@
...
-  # Automatically mount a ServiceAccount's API credentials?
+  # -- Automatically mount a ServiceAccount's API credentials?
   automount: true
-  # Annotations to add to the service account
+  # -- Annotations to add to the service account
   annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
+  # -- The name of the service account to use.
+  # -- If not set and create is true, a name is generated using the fullname template
```
Comment (simisimis) [inline liminal-manual/values.yaml:31]: see above multi-line comment

File: minametrix/values.yaml [hunk 4]
```diff
@@ -2,32 +2,41 @@
...
-  # Specifies whether a service account should be created
+  # -- Specifies whether a service account should be created
   create: true
-  # Annotations to add to the service account
+  # -- Annotations to add to the service account
   annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
+  # -- The name of the service account to use.
+  # -- If not set and create is true, a name is generated using the fullname template
```
Comment (simisimis) [inline minametrix/values.yaml:29]: see above multi-line comment

File: node-stats-collector/values.yaml [hunk 5]
```diff
@@ -1,30 +1,38 @@
...
-  # Specifies whether a service account should be created
+  # -- Specifies whether a service account should be created
   create: true
-  # Annotations to add to the service account
+  # -- Annotations to add to the service account
   annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
+  # -- The name of the service account to use.
+  # -- If not set and create is true, a name is generated using the fullname template
```
Comment (simisimis) [inline node-stats-collector/values.yaml:25]: see above multi-line comment

File: pgt-leader-bot/values.yaml [hunk 6]
```diff
@@ -2,66 +2,91 @@
...
-  # Automatically mount a ServiceAccount's API credentials?
+  # -- Automatically mount a ServiceAccount's API credentials?
   automount: true
-  # Annotations to add to the service account
+  # -- Annotations to add to the service account
   annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
+  # -- The name of the service account to use.
+  # -- If not set and create is true, a name is generated using the fullname template
```
Comment (simisimis) [inline pgt-leader-bot/values.yaml:75]: see above multi-line comment

File: delegation-verify-coordinator/values.yaml [hunk 7]
```diff
@@ -55,33 +91,40 @@ coordinator:
...
+
+  # -- Specifies whether a service account should be created
   create: true
-  # Automatically mount a ServiceAccount's API credentials?
+  # -- Automatically mount a ServiceAccount's API credentials?
   automount: true
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
+  # -- The name of the service account to use.
+  # -- If not set and create is true, a name is generated using the fullname template
```
Comment (simisimis) [inline delegation-verify-coordinator/values.yaml:113]: nitpick: I think this line is ignored and only the last comment line is picked up. So potentially we either merge the comment lines or remove one of them

**Review by simisimis — APPROVED** [review]
other than a repeating non blocking comment this is a very nice chore overtaking!!!
:clap:

File: uptime-service-payloads-scrapper/values.yaml (hunk 1 above)
Comment (kaozenn) [reply uptime-service-payloads-scrapper/values.yaml:27]: This should be fixed now

File: node-stats-collector/values.yaml (hunk 5 above)
Comment (kaozenn) [reply node-stats-collector/values.yaml]: This should be fixed now

File: delegation-verify-coordinator/values.yaml (hunk 7 above)
Comment (kaozenn) [reply delegation-verify-coordinator/values.yaml]: This should be fixed now

File: pgt-leader-bot/values.yaml (hunk 6 above)
Comment (kaozenn) [reply pgt-leader-bot/values.yaml]: This should be fixed now

File: minametrix/values.yaml (hunk 4 above)
Comment (kaozenn) [reply minametrix/values.yaml]: This should be fixed now

File: mina-rosetta/values.yaml (hunk 2 above)
Comment (kaozenn) [reply mina-rosetta/values.yaml]: This should be fixed now

**Pull request merged**